### DISCUSSION TITLE
Why not import { useTranslation } from 'next-translate' ?

### DISCUSSION THREAD

**Discussion by ffan0811** [vinissimus/next-translate#714] [Q&A] [answered]
This question might not be related to next-translate specifically.

but

is there reason why next-translate imports like this
`import useTranslation from 'next-translate/useTranslation'` ?
not `import { useTranslation } from 'next-translate'` ?

**Comment by aralroca** [comment, marked as answer]
It is the easiest way to import only what you need. Even if webpack does treeshaking it ends up taking more kb than it should.

**Reply by ffan0811** [reply]
I thought there weren't differences between named import and default import when it comes to efficiency. Thank you for the answer!
